### PR TITLE
minor fix to capitalization

### PR DESCRIPTION
**Pull request by codeboten**
Traces doesn't need to be capitalized everywhere.

### CODE REVIEW

**Comment by avillela** [comment]
@codeboten While you're at it, can you replace this line:

```
- Sign up for one of our monthly interview/feedback sessions
```

with this:

```
- [Sign up for one of our monthly interview/feedback sessions](https://opentelemetry.io/community/end-user/interviews-feedback/)
```

I tried to do an in-line suggestion, but for some reason, it wouldn't let me? 🤷‍♀️

**Comment by codeboten** [comment]
@avillela done!

File: content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md [hunk 1]
```diff
@@ -333,7 +333,7 @@ organization, we’d love to hear from you! Ways to share:
 - Join our monthly
   [End Users Discussion Group calls](/community/end-user/discussion-group/)
 - [Join our OTel in Practice sessions](/community/end-user/otel-in-practice/)
-- Sign up for one of our monthly interview/feedback sessions
+- [Sign up for one of our monthly interview/feedback sessions](https://opentelemetry.io/community/end-user/interviews-feedback/)
```
Comment (codeboten) [inline content/en/blog/2023/otel-end-user-q-and-a-series-otel-and-graphql.md:336]:
```suggestion
- [Sign up for one of our monthly interview/feedback sessions](/community/end-user/interviews-feedback/)
```

**Review by chalin — APPROVED** [review]
Thanks!

**Pull request merged**